src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -54,10 +61,9 @@ const SampleChatScreen = () => {
     goBackFromChatScreen,
     goToChatInfoScreen,
     sendChat,
-    updateChatContinuity,
-    loadingChat
+    updateChatContinuity
   } = useChatScreenHook(ANONYMOUS);
-
+  const {replyPreview, clearReplyPreview} = useMessageHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useMessageHook()` is introduced here to get `replyPreview` and `clearReplyPreview`. This seems fine as long as the hook is implemented correctly and provides the expected functionality.



<!-- This is an auto-generated comment by OSS CodeRabbit -->